### DISCUSSION TITLE
how to read/write Panda dataframes?

### DISCUSSION THREAD

**Discussion by miaxhee** [cozodb/cozo#16] [Q&A] [answered]
Hi,

have an other question.

For the python install you wrote"pip install "pycozo[embedded,pandas]",
or pip install "pycozo[embedded]" if you don't want **[Pandas dataframe] support**"

Q: How du I get Panada support? I mean, how can I read from Panda dataframes or write to Panda dataframes?

**Comment by zh217** [comment, marked as answer]
If you followed the instruction, then you should get back dataframe automatically when you query.

There is no clear-cut way of writing dataframes into the database. It doesn't work that way.

**Reply by miaxhee** [reply]
OK, thanks! At first I didn't saw the additional instructions behind the pycoso link. This helped and now it works. Thank you.
